tests/fixtures/scenarios/custom-command/atmos.yaml

```diff
@@ -0,0 +1,35 @@
+base_path: "./"
+
+components:
+  terraform:
+    base_path: "components/terraform"
+    apply_auto_approve: false
+    deploy_run_init: true
+    init_run_reconfigure: true
+    auto_generate_backend_file: false
+  
+stacks:
+  base_path: "stacks"
+  included_paths:
+    - "deploy/**/*"
+  excluded_paths:
+    - "**/_defaults.yaml"
+  name_pattern: "{stage}"
```

Comment:
We're not testing stacks
```suggestion
```

Reply:
apparently we can not remove `stacks.base_path` and `stacks.included_paths`.
According to me this is a bug and i should fix it. Please let me know if this behavior is expected.

Reply:
Yes, please fix this in a new PR.

Reply:
Will be fixed in: https://linear.app/cloudposse/issue/DEV-3041/bug-fix-custom-commands-require-stacksbase-path-and-stacksincluded